jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/DatePickersMonthTests.java

```diff
@@ -70,7 +67,7 @@ public class DatePickersMonthTests extends TestsInit {
     private String chosenMonth = chosenMonthFull.substring(0, 3);
     private String chosenMonthTwo = chosenMonthTwoFull.substring(0, 3);
 
```

Comment:
Please, add descriptions to the tests if they are missing

Reply:
Added description to test.